packages/fluentui/docs/src/app.tsx

```diff
@@ -23,8 +23,7 @@ import Routes from './routes';
 // Experimental dev-time accessibility attributes integrity validation.
 import { setup } from '@fluentui/ability-attributes';
 
-// Temporarily disabling the validation for Screener.
-if (process.env.NODE_ENV !== 'production' && !process.env.SCREENER) {
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
Not totally sure what this does so didn't remove the entire check and only removed the Screener one.

Reply:
@TristanWatanabe this adds a panel with validation warnings. We don't want to have it to be visible in production builds anyway, so let's keep it.